### PR TITLE
All Sipity functional test scenarios

### PR DESCRIPTION
**Pull request by NicHaydel95**
Tested on pre-prod. I tried to keep one of the test users from having
done the first time login so that it can be tested later.  Viewing
submitted etd's results in an error because it does not keep the user
logged in.

### CODE REVIEW

File: spec/sipity/functional/etd_spec.rb [hunk 1]
```diff
@@ -1,20 +1,86 @@
...
-    end
+
+feature 'User Browsing', js: true do
+  scenario 'Visit Homepage' do
+    visitHome
+  end
+
+  scenario 'FIRST TIME User Sign In', js: true do
+    signIn
+    welcome_page = Curate::Pages::ETDWelcomePage.new
```
Comment (jeremyf) [inline spec/sipity/functional/etd_spec.rb]: I'm thinking these are in the wrong namespace. Shouldn't it be `Sipity::Pages::ETDWelcomePage`?

File: spec/sipity/pages/cas_login_page.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require 'csv'
+module Curate
```
Comment (jeremyf) [inline spec/sipity/pages/cas_login_page.rb]: Rename to `module Sipity`

File: spec/sipity/pages/dashboard.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+module Curate
```
Comment (jeremyf) [inline spec/sipity/pages/dashboard.rb]: Rename to `module Sipity`

File: spec/sipity/pages/etd_submission_page.rb [hunk 4]
```diff
@@ -0,0 +1,40 @@
+module Curate
```
Comment (jeremyf) [inline spec/sipity/pages/etd_submission_page.rb]: Rename to `module Sipity`

File: spec/sipity/pages/etd_welcome.rb [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+module Curate
```
Comment (jeremyf) [inline spec/sipity/pages/etd_welcome.rb]: Rename to `module Sipity`

File: spec/sipity/pages/home_page.rb [hunk 6]
```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+module Curate
```
Comment (jeremyf) [inline spec/sipity/pages/home_page.rb]: Rename to `module Sipity`

File: spec/sipity/pages/new_deposit.rb [hunk 7]
```diff
@@ -0,0 +1,40 @@
+module Curate
```
Comment (jeremyf) [inline spec/sipity/pages/new_deposit.rb]: Rename to `module Sipity`

File: spec/sipity/pages/signed_in_page.rb [hunk 8]
```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+module Curate
```
Comment (jeremyf) [inline spec/sipity/pages/signed_in_page.rb]: Rename to `module Sipity`

File: spec/sipity/pages/submitted_etd.rb [hunk 9]
```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+module Curate
```
Comment (jeremyf) [inline spec/sipity/pages/submitted_etd.rb]: Rename to `module Sipity`

File: spec/sipity/functional/etd_spec.rb [hunk 10]
```diff
@@ -1,20 +1,86 @@
...
+  end
+
+  scenario 'View Dashboard', js: true do
+    returnSignIn
+    find_link("Dashboard").click
+    dashboard = Sipity::Pages::Dashboard.new
+    expect(dashboard).to be_on_page
+  end
+
+  scenario 'Start and ETD Submission', js: true do
```
Comment (jeremyf) [inline spec/sipity/functional/etd_spec.rb]: Change "Start and" to "Start an"

File: spec/sipity/functional/etd_spec.rb [hunk 11]
```diff
@@ -1,20 +1,86 @@
...
+    find_link('Sign in').click
+  end
+  casLogin = Sipity::Pages::CASLoginPage.new(current_logger )
+  expect(casLogin).to be_on_page
+  casLogin.completeLogin
+  sleep(6)
+end
+
+
+def returnSignIn
```
Comment (jeremyf) [inline spec/sipity/functional/etd_spec.rb]: I'm not certain what this method means. Either update the method name to better convey the intention or add documentation. I prefer an updated method name.

File: spec/sipity/pages/dashboard.rb [hunk 12]
```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+module Sipity
+  module Pages
+    class Dashboard
+      include Capybara::DSL
+      include CapybaraErrorIntel::DSL
+
+      def on_page?
+          #on_valid_url? &&
```
Comment (jeremyf) [inline spec/sipity/pages/dashboard.rb]: Remove commented out code.

File: spec/sipity/pages/etd_submission_page.rb [hunk 13]
```diff
@@ -0,0 +1,40 @@
+module Sipity
+  module Pages
+    class ETDSubmissionPage
+      include Capybara::DSL
+      include CapybaraErrorIntel::DSL
+
+      def on_page?
+          #on_valid_url? &&
```
Comment (jeremyf) [inline spec/sipity/pages/etd_submission_page.rb]: Remove commented out code

File: spec/sipity/pages/etd_submission_page.rb [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+          status_response_ok? &&
+          valid_page_content?
+      end
+
+      def on_valid_url?
```
Comment (jeremyf) [inline spec/sipity/pages/etd_submission_page.rb]: Is this method used anymore? If not, remove it.

File: spec/sipity/pages/new_deposit.rb [hunk 15]
```diff
@@ -0,0 +1,40 @@
+module Sipity
+  module Pages
+    class NewDepositPage
+      include Capybara::DSL
+      include CapybaraErrorIntel::DSL
+
+      def on_page?
+          #on_valid_url? &&
```
Comment (jeremyf) [inline spec/sipity/pages/new_deposit.rb]: Remove commented out code.

File: spec/sipity/pages/new_deposit.rb [hunk 16]
```diff
@@ -0,0 +1,40 @@
...
+          status_response_ok? &&
+          valid_page_content?
+      end
+
+      def on_valid_url?
```
Comment (jeremyf) [inline spec/sipity/pages/new_deposit.rb]: Is this needed anymore? If not, remove.

File: spec/sipity/pages/dashboard.rb [hunk 17]
```diff
@@ -0,0 +1,35 @@
...
+          status_response_ok? &&
+          valid_page_content?
+      end
+
+      def on_valid_url?
```
Comment (jeremyf) [inline spec/sipity/pages/dashboard.rb]: Is this method used anymore? If not, remove it.

**Comment by NicHaydel95** [comment]
@jeremyf is there anything else you wanted done before merging?

File: spec/sipity/pages/etd_submission_page.rb (hunk 14 above)
Comment (jeremyf) [inline spec/sipity/pages/etd_submission_page.rb]: @NicHaydel95 Thoughts on this method?

File: spec/sipity/pages/new_deposit.rb (hunk 16 above)
Comment (jeremyf) [inline spec/sipity/pages/new_deposit.rb]: @NicHaydel95 Thoughts on this method?

**Comment by mhallwac** [comment]
@jeremyf it seems like these changes have already been made
https://github.com/ndlib/QA_tests/commit/a5064d8d6a3317020850351d77d0d54b04f5ade4

**Pull request merged**